src/rahul_Hiremath/assignment_26/Ass_26_1.java

```diff
@@ -0,0 +1,36 @@
+/*Program 1: Write a method which will return a string containing maximum digits.
+Input : {"P1u2r2va", "1K2omal", "9S4hivan4i4", "1K2r3ati", "4M4a5d5h1v1i93"}
+Output : 4M4a5d5h1v1i93*/
+
+package rahul_Hiremath.assignment_26;
+
+public class Ass_26_1 {
+
+	String maxStringHavingMaxDigits(String[] str) {
+		int countFirstString = 0;
+		int count = 0;
+		String maxDigitString = null;
+		String[] output = new String[str.length];
+		for (int i = 0; i < output.length; i++) {
+			for (int j = 0; j < str[i].length(); j++) {
```

Comment:
count should be init with 0 for new word right, your logic break for below input

String [] arr={"P1u2r2va", "1K2omal", "4M4a5d5h1v1i93234","9S4hivan4i4", "1K2r3ati", "4M4a5d5h1v1i93"};

Reply:
My bad I missed count 0 condition after loop